environments/docker/packer.json

```diff
@@ -129,9 +129,9 @@
       "inline": [
 	       "echo nbd | sudo tee -a /etc/modules",
 	       "echo options nbd nbds_max=1024 | sudo tee /etc/modprobe.d/nbd.conf",
-	       "sudo curl -sSL https://github.com/storageos/go-cli/releases/download/0.10.0/storageos_linux_amd64 -o /usr/local/bin/storageos",
+	       "sudo curl -sSL https://github.com/storageos/go-cli/releases/download/1.0.0/storageos_linux_amd64 -o /usr/local/bin/storageos",
 	      "sudo chmod +x /usr/local/bin/storageos",
-        "sudo docker pull storageos/node:0.10.0",
+        "sudo docker pull storageos/node:1.0.0",
```

Comment:
Suggest we bump everything to 1.0.2 since we know it has Katacoda specific fixes.

Reply:
I don't know whether this can/should be bumped. @Arau do you know?